### ISSUE TITLE
error in iOS Simulator 6, 6plus,but other Simulator and iPhone 4s 4 are ok

### ISSUE THREAD

**Issue by muyushifang07** [Zedenem/XMLParser#1]
"_OBJC_CLASS_$_GHAsyncTestCase", referenced from:
      _OBJC_CLASS_$_XMLParserTests in XMLParserTests.o
  "_OBJC_METACLASS_$_GHAsyncTestCase", referenced from:
      _OBJC_METACLASS_$_XMLParserTests in XMLParserTests.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

hope to help solve this issue！thank you

**Comment by gaoyuanwang** [comment]
I met the same problem.Do you have an idea now?

**Comment by muyushifang07** [comment]
I forget the demo when I test the xml-message-parsering ,but now what I used is OK!I don*t which one !if your problem is still there！You can contact me ！

**Comment by Zedenem** [comment]
Hi @muyushifang07,

So, the issue is with the demo project? If so, I will try and update it ASAP.

Best,

**Comment by muyushifang07** [comment]
NO,Actually,It appears when I  put your some files into my test project !Now ,You can also try it as same!
